etc/docker/php/entrypoint.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
+chown -R www-data:www-data -R app/cache app/logs
```

Comment:
app/cache and app/logs are stored in Docker volumes, so nothing changes on your host